core/src/main/java/com/novoda/accessibility/ActionsMenuAlertDialog.java

```diff
@@ -0,0 +1,24 @@
+package com.novoda.accessibility;
+
+import android.content.Context;
+import android.support.v7.app.AlertDialog;
+import android.view.Menu;
+import android.view.MenuItem;
+
+public class ActionsMenuAlertDialog {
+
+    public static AlertDialog.Builder create(Context context, Menu menu, MenuItem.OnMenuItemClickListener menuItemClickListener) {
```

Comment:
returning the builder instead gives the consumer a chance to set the title or not, and other things if they want.